docs/02.API-REFERENCE.md

```diff
@@ -391,6 +391,19 @@ An opaque declaration of the JerryScript context structure.
 typedef struct jerry_context_t jerry_context_t;
 ```
 
+*New in version [[NEXT_RELEASE]]*.
```

Comment:
This seems to be at an incorrect place. It should be under the new enum type (after the description)